testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -277,15 +275,15 @@ impl BlockMinerThread {
 
     #[cfg(test)]
     fn fault_injection_block_announce_stall(new_block: &NakamotoBlock) {
-        if TEST_BLOCK_ANNOUNCE_STALL.get() {
+        if relay::fault_injection::stacks_announce_is_blocked() {
```

Comment:
These seem like they could be independently controlled... `TEST_BLOCK_ANNOUNCE_STALL` is used in the integration tests to prevent the miner themselves from announcing a new block (this is used to test that the signer set can announce blocks by themselves) -- but if the relayer stall is also active, it seems like it prevents the chains coordinator thread from waking up?

Reply:
At the time this PR was written, this was an attempt to make some CI tests less flaky.  It appears that not only is this not necessary anymore, but also my code here has led to CI breakage.